solidity/scripts/multiply-local-eth-btc-price.js

```diff
@@ -0,0 +1,29 @@
+const ETHBTCPriceFeedMock = artifacts.require("ETHBTCPriceFeedMock")
+
+async function run() {
+  try {
+    if (process.argv.length <= 4 || process.argv[4] === "help") {
+      console.log(
+        "run `truffle exec multiply-local-eth-btc-price.js <factor>` to update the current price by a factor of <factor>",
```

Comment:
Help output tends to be _reasonably_ standard for tools, and would look a little more like this:

```
truffle exec multiply-local-eth-btc-price.js <factor>
    Updates the local mock ETHBTC price by the specified factor.

    <factor>
        A factor to multiply the current ETHBTC price by. ETHBTC price
        is in wei per satoshi, so a factor of 2 means the price of one satoshi
        has doubled in terms of wei, i.e. it takes twice as many wei to buy
        one satoshi. In terms of collateralization, this means a deposit that
        was 100% collateralized is now 50% collateralized.
```